kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiDatabaseMetaData.java

```diff
@@ -122,8 +126,50 @@ public ResultSet getCatalogs() throws SQLException {
         .build();
   }
 
+  private static final class ClientInfoPropertiesResultSet extends KyuubiMetaDataResultSet<Object> {
+    private static final String[] COLUMNS = {"NAME", "MAX_LEN", "DEFAULT_VALUE", "DESCRIPTION"};
+    private static final String[] COLUMN_TYPES = {"STRING", "INT", "STRING", "STRING"};
+
+    private static final Object[][] DATA = {
+      {"ApplicationName", 1000, null, null},
+      // Note: other standard ones include e.g. ClientUser and ClientHostname,
```

Comment:
Are you sure that the `DATA` of Hive doesn't need the ClientUser and ClientHostname? If yes, please remove the note because this has nothing to do with other standards.

Reply:
@yangrong688, any feedback?

Reply:
Thanks for your comment. Currently, Hive does not need ClientUser and ClientHostname, I will remove the note.